released/packages/coq-mathcomp-word/coq-mathcomp-word.2.2/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "pierre-yves@strub.nu"
+
+homepage: "https://github.com/jasmin-lang/coqword"
+bug-reports: "https://github.com/jasmin-lang/coqword/issues"
+dev-repo: "git+https://github.com/jasmin-lang/coqword.git"
+license: "MIT"
+
+synopsis: "Yet Another Coq Library on Machine Words"
+
+build: [ "dune" "build" "-p" name "-j" jobs ]
+depends: [
+  "dune" {>= "2.8"}
+  "coq" {>= "8.12"}
+  "coq-mathcomp-ssreflect" {>= "1.12" & < "1.19~"}
+  "coq-mathcomp-algebra"
+]
+tags: [
+  "category:Computer Science/Data Types and Data Structures"
+  "keyword:machine words"
+  "logpath:mathcomp.word"
+  "date:2023-11-21"
+]
+authors: ["Pierre-Yves Strub"]
+
+url {
+  src: "https://github.com/jasmin-lang/coqword/archive/refs/tags/v2.2.tar.gz"
```

Comment:
```suggestion
  src: "https://github.com/jasmin-lang/coqword/archive/refs/tags/v2.2.tar.gz"
  checksum: "sha512=a0f2802d3bf670049b29e9f061d87b34637a17e054438e0a402ea01308d57a9bef6d6a3fc6b43e210127682e0411c613d857f808564c6923e686430aac31fc25"
```

Reply:
Why? github is free to update that file; that would make the hash spuriously invalid.

Reply:
Sure, but we have the convention that we add hashes for the current archive, this can be used for local validation and caching and the like. More generally, if there isn't a decision saying otherwise, I view having a hash as a requirement for merge.

Reply:
Even a random hash? Seems weird…

Reply:
@vbgl if you want checksums to be meaningful, why not manually put an archive into the release as a downloadable file? In this case, to my knowledge, GitHub guarantees it to be stable over time. Enrico has automated this for Coq-Elpi releases, for example: https://github.com/LPCIC/coq-elpi/releases/tag/v1.19.3